<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Target</class>
 <widget class="QWidget" name="Target">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1024</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Target</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(31, 31, 31);</string>
  </property>
  <widget class="QScrollArea" name="TargetScroll">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>71</y>
     <width>1920</width>
     <height>951</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="TargetScrollWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>951</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1920</width>
      <height>951</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1920</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>2034</width>
       <height>1001</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="TargetLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>540</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>540</number>
      </property>
      <item>
       <widget class="QLabel" name="TargetLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>880</width>
          <height>250</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>48</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(143, 115, 101);
background-color: rgba(0, 0, 0, 0);
padding-top: 60px;
padding-bottom: 60px;</string>
        </property>
        <property name="text">
         <string>Target</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="TargetList">
        <property name="spacing">
         <number>30</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="bottomMargin">
         <number>40</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>10</number>
          </property>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="NewTargetButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>36</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(&quot;src/assets/TargetNewButton.png&quot;);
border: none;
background-repeat: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="NewTargetLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLineEdit" name="InputNewTarget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>710</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>710</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">padding-left: 10px;
padding-right: 10px;
padding-top: 8px;
padding-bottom: 8px; 
border: 0px;
border-radius: 10px;
background-color: rgb(40, 40, 40);
color: rgb(129, 129, 129)
</string>
          </property>
          <property name="placeholderText">
           <string>Enter new target...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="InputYearTarget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>85</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>85</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">padding-left: 10px;
padding-right: 10px;
padding-top: 8px;
padding-bottom: 8px; 
border: 0px;
border-radius: 10px;
background-color: rgb(40, 40, 40);
color: rgb(129, 129, 129)
</string>
          </property>
          <property name="placeholderText">
           <string>2022...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SaveTargetButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>58</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>58</width>
            <height>50</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(&quot;src/assets/TargetSaveButton.png&quot;);
border: none;
background-repeat: none;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="ReachedLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>250</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>250</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(190, 190, 190);
background-color: rgba(0, 0, 0, 0);
padding-top: 100px;
padding-bottom: 15px;</string>
        </property>
        <property name="text">
         <string>Reached</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="ReachedList">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
